api/pkg/apis/v1alpha1/managers/instances/instances-manager.go

```diff
@@ -47,6 +47,7 @@ func (t *InstancesManager) DeleteState(ctx context.Context, name string, namespa
 	})
 	var err error = nil
 	defer observ_utils.CloseSpanWithError(span, &err)
+	defer observ_utils.EmitUserDiagnosticsLogs(ctx, &err)
```

Comment:
Just curious, do we care about the order of logs emit and close span?

Reply:
No, I am not realize there's any case we need to enforce the order?